scripts/drone/events/cron.star

```diff
@@ -24,6 +24,17 @@ def cronjobs():
         grafana_com_nightly_pipeline(),
     ]
 
+def authenticate_gcr_step():
+    return {
+        "name": "authenticate-gcr",
+        "image": "docker:dind",
```

Comment:
Does this launch its own Docker instance? If so, why do you need to mount the docker socket? Otherwise, shouldn't a docker-cli image be enough here?

Reply:
It should be enough, but that's what we use for other ops as well and I thought it would be easier to just use this one. Can change if you have a strong preference, although I don't think it's a big issue.

Reply:
Ah ok, didn't check. OK for me :)